### ISSUE TITLE
Add a way to reinstall the driver with one command

### ISSUE THREAD

**Issue by zurk** [bblfsh/bblfshd#197]
To reinstall python driver I need to remove it first and install the new one after. Good to have something like `reinstall` or `install -f` subcommand for it.

**Comment by juanjux** [comment]
Actually, a `-f` switch was recently added by @dennwc for the `install` command but I'm not sure if it will reinstall the driver or just ignore the error, @dennwc can you clarify?

**Comment by dennwc** [comment]
No, it will just skip "already installed" errors. I propose to add an `-u` (`--update`) flag instead. Or `-r` (`--replace`).

**Comment by zurk** [comment]
yes, `--update` is totally fine :)
